tests/pipelines/main.nf.test

```diff
@@ -150,79 +150,6 @@ nextflow_pipeline {
         }
     }
 
-    test("Full pipeline hashes and missing data") {
```

Comment:
These tests were moved to the file `tests/pipelines/main_missing_thresholds.nf.test`